src/main/java/com/onepassword/burpanalyzer/ui/OnePasswordSessionTab.java

```diff
@@ -149,17 +142,14 @@ public void setMessage(byte[] content, boolean isRequest) {
     private void updateDecryptedPayload(final byte[] sessionKey, final EncryptedMessage encrypted) {
         final var decrypted = encrypted.decrypt(sessionKey);
 
-        final var text = decrypted.fold(
-                DecryptionError::getReadableError,
-                dm -> {
-                    final var body = dm.getBody();
-                    if(body != null && body.length > 0) {
-                        return helpers.bytesToString(body);
-                    } else {
-                        return "";
-                    }
-                }
-        );
+        final var text = decrypted.checkResult().map( dm -> {
+            final var body = dm.getBody();
+            if(body != null && body.length > 0) {
```

Comment:
Nit: Spacing

Reply:
Good catch - made that consistent everywhere.